common/index.jsx

```diff
@@ -11,6 +11,10 @@ import { Root } from 'components';
 import { Routing, history } from './routing';
 import { store } from './store';
 
+import u2f from './vendor/u2f-api';
+
+window.u2f = u2f;
```

Comment:
Included because `ledger3.js` expects `u2f` to be available globally. We could import it directly into the library file, though `ledger3.js` is licensed under `Apache 2.0`. Not sure what the deal is there.

Reply:
tldr legal says it's kosher to edit: https://tldrlegal.com/license/apache-license-2.0-(apache-2.0)

I did the same to digital bitbox, https://github.com/MyEtherWallet/MyEtherWallet/blob/digital-bitbox/common/vendor/digital-bitbox-usb.js#L11

Reply:
Well that's a super helpful website! :smile:  

Good to know the global can be avoided, I'll make the change.